examples/egl_common.c

```diff
@@ -0,0 +1,168 @@
+#define _POSIX_C_SOURCE 200809L
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <unistd.h>
+#include <wayland-client.h>
+#include <wlr/util/log.h>
+
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+
+#include "egl_common.h"
+
+struct egl_ctx *egl_context_create(struct wl_display *display, 
+		const EGLint *config_attribs) {
+	struct egl_ctx *egl = calloc(1, 
+		sizeof(struct egl_ctx));
```

Comment:
Style: some lines like this one are split but they are under the 80-char limit. Please un-fold these.

Reply:
Done! Let me know if you want me to fix anything else.